src/blueprint/azext_blueprint/_params.py

```diff
@@ -0,0 +1,215 @@
+        c.argument('artifact_name', help='A unique name of this resource group artifact.')
+        c.argument('display_name', help='Display name of this resource group artifact.')
+        c.argument('description', help='Description of the blueprint artifact.')
+        c.argument('depends_on', nargs='+', help='Artifacts which need to be deployed before the specified artifact.')
+        c.argument('tags', tags_type, help='Tags to be assigned to this resource group.')
+
+    with self.argument_context('blueprint resource-group update') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('rg_name', help='Name of this resource group. Leave empty if the resource group name will be specified during the blueprint assignment.')
+        c.argument('rg_location', help='Location of this resource group. Leave empty if the resource group location will be specified during the blueprint assignment.')
+        c.argument('artifact_name', help='A unique name of this resource group artifact.')
+        c.argument('display_name', help='Display name of this resource group artifact.')
+        c.argument('description', help='Description of the blueprint artifact.')
+        c.argument('depends_on', nargs='+', help='Artifacts which need to be deployed before the specified artifact.')
+        c.argument('tags', tags_type, arg_group='Resource Group', help='Tags to be assigned to this resource group.')
+
+    with self.argument_context('blueprint resource-group delete') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('artifact_name', help='A unique name of this resource group artifact.')
+
+    with self.argument_context('blueprint resource-group show') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('artifact_name', help='A unique name of this resource group artifact.')
+
+    with self.argument_context('blueprint resource-group list') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+
+    with self.argument_context('blueprint artifact policy create') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('artifact_name', help='Name of the blueprint artifact.')
+        c.argument('display_name', help='DisplayName of this artifact.')
+        c.argument('description', help='Description of the blueprint artifact.')
+        c.argument('depends_on', nargs='+', help='Artifacts which need to be deployed before the specified artifact.')
+        c.argument('policy_definition_id', help='Azure resource ID of the policy definition.')
+        c.argument('resource_group_art', help='Name of the resource group artifact to which the policy will be assigned.')
+        c.argument('parameters', arg_type=parameter_type)
+
+    with self.argument_context('blueprint artifact policy update') as c:

```

Comment:
update command doesn't need to specify policy_definition_id when doing the operation?

Reply:
No, it's also an artifact. So it uses `artifact_name` to identify the resource to update.